pkg/ip/link.go

```diff
@@ -39,37 +39,61 @@ func makeVeth(name, peer string, mtu int) (netlink.Link, error) {
 	return veth, nil
 }
 
+func makeVeth(name string, mtu int) (peerName string, veth netlink.Link, err error) {
+	for {
```

Comment:
Yeah the chance now might be minuscule but in anticipation of something changing or a bug elsewhere, etc etc (especially as this calls other functions)